test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1995/config/form';
+
+describe('Edu 1995 schoolSelection', () => {
```

Comment:
I thought about that one, but isn't that going away? Or was what I'm thinking the bit where we conditionally add an option to the education type?